packages/server/src/main/java/software/uncharted/terarium/hmiserver/models/dataservice/regnet/RegNetBaseProperties.java

```diff
@@ -0,0 +1,23 @@
+package software.uncharted.terarium.hmiserver.models.dataservice.regnet;
+
+import lombok.Data;
+import lombok.experimental.Accessors;
+import software.uncharted.terarium.hmiserver.annotations.TSModel;
+import software.uncharted.terarium.hmiserver.annotations.TSOptional;
+import software.uncharted.terarium.hmiserver.models.dataservice.modelparts.ModelGrounding;
+
+import java.util.List;
+
+@Data
+@Accessors(chain = true)
+@TSModel
+public class RegNetBaseProperties {
+	@TSOptional
+	private String name;
```

Comment:
There is no `id` because of the conclusion from today's TA2 WG?

Reply:
There is no `id` because I don't think base-properties are (or should be) uniquely identifiable, and the schema itself has no notion of id of this type of object.

But I think think the TSOptional annotations here are correct so I will fix that.